### ISSUE TITLE
Training problem about simplefold

### ISSUE THREAD

**Issue by Paulie-ai** [apple/ml-simplefold#44]
Hi, I am trying to reproduce SimpleFold. I noticed that the residues generated using my weights have severe steric clashes (atoms are clumped together). My training MSE loss is around 0.3 before fine-tuning(lddt_loss: 0.23 and mse loss 0.026 ), after fine-tuning the lddt_loss was increased to ~0.5-0.7.  Any advice on how to fix this ?

**Comment by Paulie-ai** [comment]
Thanks for your reply. I rewrote the simplefold training and inference scripts with pytorch_lightening, and the configuration filde is same as the original. The raw weight work fine when tested with my inference code. Therefore, could the issue be related to the batch size/the dataset size/epochs used for training?

**Comment by yuyangw** [comment]
Yes, it could be related to batch size / epochs. How are these hyper-parameters set in your case?
